serverless.yml

```diff
@@ -9,12 +9,16 @@ plugins:
 provider:
   name: aws
   runtime: nodejs8.10
-  # region: ap-southeast-2
+  region: ap-southeast-2
   iamRoleStatements:
     - Effect: "Allow"
-      Action: 
-       - "cloudwatch:DescribeAlarmHistory"
-       - "cloudwatch:PutMetricData"
+      Action:
+        - "cloudwatch:DescribeAlarmHistory"
+        - "cloudwatch:PutMetricData"
+      Resource: "*"
+    - Effect: "Allow"
+      Action:
+        - events:*
```

Comment:
Again - does this need to be read/write- can it be read only?

Is there a managed policy that would be more appropriate?

Reply:
Made it more specific